docs/advanced/os-dependent_libraries.md

```diff
@@ -140,9 +136,9 @@ bindir = os.path.dirname(os.path.realpath(os.path.dirname(__file__)))
 libdir = os.path.join(bindir, "lib")
 platform = sys.platform
 if platform.startswith("linux"):
-	sys.path.insert(0, os.path.join(libdir, "3rdparty/linux_with_deps"))
-	sys.path.insert(0, os.path.join(libdir, "3rdparty/linux"))
+ sys.path.insert(0, os.path.join(libdir, "3rdparty/linux_with_deps"))
+ sys.path.insert(0, os.path.join(libdir, "3rdparty/linux"))
 if platform.startswith("win"):
-	sys.path.insert(0, os.path.join(libdir, "3rdparty/windows"))
+ sys.path.insert(0, os.path.join(libdir, "3rdparty/windows"))
```

Comment:
Should we change indentation here?

Reply:
true looks weird, removed the validation check for code blocks

Reply:
there was no reason probably, 1 tab is fine